frontend/packages/rhoas-plugin/src/models/rhoas.ts

```diff
@@ -0,0 +1,41 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const ManagedKafkaRequestModel: K8sKind = {
+  apiGroup: 'rhoas.redhat.com',
+  apiVersion: 'v1alpha1',
```

Comment:
nit: would be good to move `rhoas.redhat.com` and `v1alpha1` as const and use below as well, WDYT?

Reply:
Great idea!

Reply:
+1